tests/testsuite/tool_paths.rs

```diff
@@ -353,6 +353,29 @@ fn custom_linker_env() {
         .run();
 }
 
+#[cargo_test]
+#[cfg(not(windows))]
```

Comment:
Would it be possible to add a test to ensure that windows doesn't emit the warning, and works with a lowercase value?